order/views.py

```diff
@@ -18,6 +29,7 @@ def get(self, request):
             "price"         : int(item.product.price),
             "quantity"      : item.quantity,
             "discount_id"   : item.product.discount_id,
+            "image_url"     : item.product.productimage_set.filter(product_id=item.product.id).first().image_url
```

Comment:
이미 해당 `product`를 참조하고 있는 `product images`를 가져오는 과정이기때문에 추가적으로 `item.product.id` 로 필터링을 걸어야할 필요가 없겠죠?!
```suggestion
            "image_url"     : item.product.productimage_set.first().image_url
```

Reply:
반영했습니다~!